specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/Job_Stop_Multiple.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "jobName": "testcontainerAppsJob0",
+    "jobExecutionName": {
+      "value": [
+        {
+          "name": "jobExecution1"
+        },
+        {
+          "name": "jobExecution2"
+        }
+      ]
+    },
+    "api-version": "2022-11-01-preview"
+  },
+  "responses": {
+    "200": {},
```

Comment:
For this there can be errors as well. e.g. some of the executions in the list were non existing. So we want to at least return a response with successfully stopped jobs and the ones that weren't successful

Reply:
@trajkobal updated this, now we are returning a list of job executions for which a stop was requested, along with the status.